document-store/src/integrationTest/java/org/hypertrace/core/documentstore/DocStoreQueryV1Test.java

```diff
@@ -2437,7 +2450,12 @@ void testUpdateSetEmptyObject(final String datastoreName) throws IOException {
 
       final CloseableIterator<Document> iterator =
           collection.bulkUpdate(
-              query, updates, UpdateOptions.builder().returnDocumentType(AFTER_UPDATE).build());
+              query,
+              updates,
+              UpdateOptions.builder()
+                  .returnDocumentType(AFTER_UPDATE)
+                  .missingDocumentStrategy(MissingDocumentStrategy.SKIP_UPDATES)
```

Comment:
We may not need to modify any of the existing integration tests. This might invite a breaking change in the clients. Could you please make sure to have a default value for `missingDocumentStrategy` if unassigned?